lib/components/user/favorite-location.js

```diff
@@ -0,0 +1,236 @@
+import React, { Component } from 'react'
+import {
+  Button,
+  DropdownButton,
+  FormGroup,
+  Glyphicon,
+  InputGroup,
+  MenuItem
+} from 'react-bootstrap'
+import FontAwesome from 'react-fontawesome'
```

Comment:
This is not generally how icons are defined in this app. We should be importing the Icon component:

```
import Icon from '../narrative/icon'
```

It looks like there are a couple of other spots where FontAwesome is used like this. We should replace those as well.

Reply:
Should I replace the `FontAwesome` instance in `NavigationBar` (mobile component)? This would happen:
![image](https://user-images.githubusercontent.com/56846598/101696983-fce53900-3a44-11eb-8243-ad22507ad318.png)

Reply:
Feel free to tweak `icon.js` if the fixedWidth prop is giving you trouble. Easiest thing might be to just pass `fixedWidth={false}` to where it's used in navbar.

Reply:
NavigationBar arrow icon is updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/eccfa5ccd2006c56e279ee185ea4f48fbbe62008.